app/src/ui/diff/code-mirror-host.tsx

```diff
@@ -7,6 +7,9 @@ import 'codemirror/addon/selection/mark-selection'
 // Autocompletion plugin
 import 'codemirror/addon/hint/show-hint'
 
+// Required for some syntax highlighting support like Docker
+import 'codemirror/addon/mode/simple'
```

Comment:
@shiftkey I don't see why this would be needed in the renderer. We use a custom-built mode for our syntax highlighting so we really shouldn't need to explicitly import anything. The dockerfile mode automatically imports the simplemode *inside of the highlighter web worker* when necessary, as do many other modes.

Could you provide some more context on this change?

Reply:
From memory I thought this was related to some styling issues, but let me pull it out and confirm it's not needed.

Reply:
Yeah, I retested the Dockerfile styles with dark and light theme and couldn't spot anything weird on 8ddf521705672e46367fbbc690d515ebd3119fbf

<img width="1257" src="https://user-images.githubusercontent.com/359239/48499299-84ccca80-e80e-11e8-9ea5-23aaa5774518.png">
<img width="1261" src="https://user-images.githubusercontent.com/359239/48499300-84ccca80-e80e-11e8-807a-af58d1209089.png">